ginga/web/pgw/Widgets.py

```diff
@@ -1784,7 +2260,20 @@ def set_widget(self, child):
         self.add_ref(child)
 
     def scroll_to_end(self, vertical=True, horizontal=False):
-        pass
+        if self._rendered:
+            app = self.get_app()
+            if vertical:
+                app.do_operation('scroll_vert', id=self.id)
```

Comment:
Why not make the operation name more explicit?

```suggestion
                app.do_operation('scroll_vertical', id=self.id)
```